docs/content/node-operation/index.mdx

```diff
@@ -1,21 +1,42 @@
 ---
-title: Hello Node Operator!
+title: Flow Node
 sidebar_title: Introduction
-description: How to run your own Flow node
+description: Introduction Flow node
 ---
 
-To run a node in the Flow network, the node must:
-* be registered with sufficient stake
-* be authorized by the service account
+The Flow node is the vital component that supports the network and enables you to participate in the Flow decentralized blockchain.
```

Comment:
Plural here as well: `Flow nodes are [...]`.

Also would remove the word decentralized, it is implied in blockchain

Reply:
fixed

Reply:
How about, 
`Flow nodes are the building blocks of the Flow blockchain`.